snapshot.yaml

```diff
@@ -15,15 +13,8 @@ packages:
   - servant-serf-0.1.0
   - servant-static-th-1.0.0.0
   - timing-convenience-0.1
-
-  # https://github.com/yesodweb/persistent/pull/1209
-  - git: https://github.com/yesodweb/persistent
-    commit: ec5f9ff9373f511d520bc8dd33cfab09db50cab9
-    subdirs:
-     - persistent
-     - persistent-mongoDB
-     - persistent-postgresql
-     - persistent-template
+  - persistent-2.12.0.0
+  - persistent-mongoDB-2.11.0.0
```

Comment:
Is it alright to use `persistent` 2.12 with `persistent-mongoDB` 2.11? Also what happened to `persistent-postgresql` and `persistent-template`?

Reply:
I wasn't 100% sure on this one, I assume the fixes were done on https://hackage.haskell.org/package/persistent-2.12.0.0 but there is no release for the other ones. `persistent-postgresql` and `persistent-template` are included in the nightly already (at least the latest matching on hackage).

Reply:
nvm looks like 2.12 was a mistake https://github.com/yesodweb/persistent/pull/1211

Reply:
> Is it alright to use persistent 2.12 with persistent-mongoDB 2.11? Also what happened to persistent-postgresql and persistent-template?

The patch was in the `persistent` package and it's the only one that has a release like @aschmois said, but it does feel weird to not have the versions matching.

Reply:
Looks like everything's already in stack's nightly, with the exception of mongo which has never landed in stackage for whatever reason.